README.md

```diff
@@ -237,6 +237,8 @@ Share your apps here! Submit a pull request!
 
 * **Quit Smoking App with Flutter and Appwrite** - A simple Counting app with user auth  [Source Code](https://github.com/Jonathanvanhaaften/flutter_appwrite_1)
 
+* A simple example App that calls an **Appwrite Cloud Function** - [Article Free and easy Cloud Functions in Dart with Appwrite — Part 1](https://medium.com/@aschilken/free-and-easy-cloud-functions-in-dart-with-appwrite-part-1-190faa3dc7bf) | [Source Code](https://github.com/schilken/appwrite_cloud_functions)
```

Comment:
Can we use a relatively more meaningful name for the application here?

Reply:
@adityaoberai Well - it's nothing more than a "simple example app" just for the article to demonstrate how an Appwrite Cloud function is created, configured, and called. What would you propose?

Reply:
In that case, since the article is the "hero" of this contribution, we can keep it as a blog here under [Appwrite Services](https://github.com/appwrite/awesome-appwrite#appwrite-services).

Reply:
@adityaoberai Maybe the best match for this App+Article is "Mobile Development". I could change the title to "A simple example App in Flutter that calls an **Appwrite Cloud Function** ... ?

Reply:
> @adityaoberai Maybe the best match for this App+Article is "Mobile Development". I could change the title to "A simple example App in Flutter that calls an **Appwrite Cloud Function** ... ?

Depends on what the primary showcase area of the article is. 
If that's the app, then yes, `Mobile Development` is fine.
Otherwise, considering the article is also titled `Free and easy Cloud Functions in Dart with Appwrite — Part 1`, I'd still prefer the `Appwrite Services` category.